app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row align-items-center">
+
+        <div class="col-4">
+                <img src=<%=user.photo%> class="user-pic" alt=<%=user.name%> >
+        </div>
+
+        <div class="card col-8 d-flex flex-column justify-content-between">
+
+
+                <div class="col-7 d-flex justify-content-center flex-column">
+                        <h2 class = "h1">
+                                <%=user.name%>
+                        </h2>
+                </div>
```

Comment:
- [ ] Let's ensure that clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)

```suggestion
                <%= link_to user_path(user.id) do %>
                  <div class="col-7 d-flex justify-content-center flex-column">
		     <h2 class = "h1">
			<%=user.name%>
		    </h2>
		</div>
    <% end %>
```